core/domain/learner_group_services.py

```diff
@@ -528,14 +584,29 @@ def invite_students_to_learner_group(
 
 def remove_invited_students_from_learner_group(
     group_id: str,
-    student_ids: List[str]
+    student_ids: List[str],
+    update_group: bool
```

Comment:
Ditto

Reply:
when `remove_invited_students_from_learner_group` function is called from inside the update learner group function, as we already update the learner group there, it is not necessary to update it again in this function.